### ISSUE TITLE
Command edit-settings raises exception

### ISSUE THREAD

**Issue by tbpassin** [leo-editor/leo-editor#3336]
Running the command `edit-settings` from the minibuffer produced this traceback:
```
Traceback (most recent call last):
  File "c:\Tom\git\leo-editor\leo\core\leoKeys.py", line 2507, in callAltXFunction
    func(event)
  File "c:\Tom\git\leo-editor\leo\core\leoGlobals.py", line 244, in commander_command_wrapper
    method(event=event)
  File "c:\Tom\git\leo-editor\leo\commands\commanderHelpCommands.py", line 40, in editOneSetting
    c, p = self, self.c.p
                 ^^^^^^
AttributeError: 'Commands' object has no attribute 'c'
```
Maybe 
`c, p = self, event.c.p`?

```
Leo 6.7.3, 6.7.3 branch, build ed0dd9b408
2023-05-07 09:06:36 -0500
Python 3.11.3, PyQt version 6.4.3
Windows 10 AMD64 (build 10.0.19044) SP0
```

**Comment by edreamleo** [comment]
@tbpassin Good catch. Keep up your great testing work.

**Comment by tbpassin** [comment]
I'm hitting a number of new bugs and quirks in connection with writing my New User's Guide.  I want to tell the reader that he can do something, but when I try it myself to make sure I understand, I notice some error or quirk.

**Comment by edreamleo** [comment]
@tbpassin Not quite the curse of the demo, but related :-) Thanks for all these bug reports.

**Comment by edreamleo** [comment]
Fixed via PR #3401.
